shap/plots/_scatter.py

```diff
@@ -71,101 +85,107 @@ def scatter(
         increase plot readability when a feature is discrete. By default, ``x_jitter``
         is chosen based on auto-detection of categorical features.
 
+    title: str, optional
+        Plot title.
+
     alpha : float
         The transparency of the data points (between 0 and 1). This can be useful to
         show the density of the data points when using a large dataset.
 
-    xmin : float or string
-        Represents the lower bound of the plot's x-axis. It can be a string of the format
-        "percentile(float)" to denote that percentile of the feature's value used on the x-axis.
+    xmin, xmax, ymin, ymax : float, string, aggregated Explanation or None
+        Desired axis limits. Can be a float to specify a fixed limit.
 
-    xmax : float or string
-        Represents the upper bound of the plot's x-axis. It can be a string of the format
-        "percentile(float)" to denote that percentile of the feature's value used on the x-axis.
+        It can be a string of the format ``"percentile(float)"`` to denote that
+        percentile of the feature's value.
 
-    ax : matplotlib Axes object
-        Optionally specify an existing matplotlib ``Axes`` object, into which the plot will be placed.
-        In this case, we do not create a ``Figure``, otherwise we do.
+        It can also be an aggregated column of a single column of an :class:`.Explanation`,
+        such as ``explanation[:, "feature_name"].percentile(20)``.
+
+    overlay: dict, optional
+        Optional dictionary of curves to overlay as line plots.
+
+        The dictionary maps a curve name to a list of (xvalues, yvalues) pairs, where
+        there is one pair for each feature to be plotted.
```

Comment:
> specify that overlay only takes up to a maximum of 3 additional curves

Good idea, done.

> it would be worth considering deprecating overlay parameter in the future

Yeah, I think that's a good idea for a future change. It's becoming apparent that this function is doing a lot of work, and the overlay functionality seems a little niche and over-complex.